orphedomos-test/pom.xml

```diff
@@ -26,9 +26,16 @@
                     <groupId>org.technologybrewery.orphedomos</groupId>
                     <artifactId>orphedomos-maven-plugin</artifactId>
                     <version>${project.version}</version>
-                    <configuration>
-                        <repoId>docker-hub</repoId>
-                    </configuration>
+                    <executions>
+                        <execution>
```

Comment:
I: Making login attempts only occur in testing during the deploy phase, for ease of testing.

Reply:
Q: May be worthwhile to provide a testing profile to always log in?  Minimal lift, but also minor value add.

Reply:
We should probably do that with the Mojo test framework - which would be good for all functionality.  Would be a good follow on task to improve and keep quality consistent.